pandas/tests/io/test_parquet.py

```diff
@@ -437,6 +437,27 @@ def test_s3_roundtrip(self, df_compat, s3_resource, pa):
         check_round_trip(df_compat, pa,
                          path='s3://pandas-test/pyarrow.parquet')
 
+    @pytest.mark.xfail(
+        is_platform_windows() or is_platform_mac(),
+        reason="reading pa metadata failing on Windows/mac"
+    )
+    def test_custom_metadata(self, pa_ge_070, df_full):
```

Comment:
Add a comment about why pyarrow>=0.7.0 is required (is it?)

Reply:
Not sure. It might work with 0.5.0.

Do you really think it is worth testing?

pyarrow is already 0.9.0 and people should rather not use versions older than 0.7.0.
In my opinion the newer versions of pandas should require at least 0.7.0
the older ones are not worth testing.

Reply:
It's worth seeing if it works.